static/css/layout.css

```diff
@@ -0,0 +1,80 @@
+
```

Comment:
Can you copy this file's contents into `static/scss/layout.scss`? This wasn't working when you started with this functionality but it should work now